src/store/index.ts

```diff
@@ -1,9 +1,21 @@
-import { createStore } from "vuex";
+import { InjectionKey } from "vue";
+import { createStore, useStore as baseUseStore, Store } from "vuex";
+import { Question } from "@/types";
+import questionsJson from "@/assets/questions.json";
 
-export default createStore({
-  state: {},
-  getters: {},
-  mutations: {},
-  actions: {},
-  modules: {},
+export interface State {
+  questions: Array<Question>;
+}
+
+export const key: InjectionKey<Store<State>> = Symbol();
+
+export const store = createStore<State>({
+  state: {
+    questions: questionsJson,
```

Comment:
You put a default? Is this really the best option? What is about error handling? I will mention this in an extra comment.